### ISSUE TITLE
On environment creation, pip not finding 'capsule-layer'

### ISSUE THREAD

**Issue by JRBobDobbs** [Akhilesh-Gogikar/MultiStockRLTrading#1]
Brilliant set of articles, Kudos. Unfortunately after running:

`conda env create -f environment.yml`

I am stuck with this error message

```
Pip subprocess error:
ERROR: Could not find a version that satisfies the requirement capsule-layer==0.5.0 (from versions: none)
ERROR: No matching distribution found for capsule-layer==0.5.0

failed

CondaEnvException: Pip failed
```
I have tried installing without specifying the version i.e `pip install capsule-layer`. 
However this is the response:
```
ERROR: Could not find a version that satisfies the requirement capsule-layer (from versions: none)
ERROR: No matching distribution found for capsule-layer
```
any ideas?

**Comment by Akhilesh-Gogikar** [comment]
Hi! Thanks for pointing this out. The Capsule layer library is installed from this implementation -> https://github.com/leftthomas/CapsuleLayer

Hence, it is not available through conda and pip package managers. The solution is to comment the capsule_layer line in the environment.yml then install this library through pip.

I have updated the repo to reflect this in the instruction. It should work now. Please let me know if it works. I will close the issue then. Cheers!

**Comment by JRBobDobbs** [comment]
Thanks for that, it fixed the issue. However, I did have a bunch of package conflict issues when importing the environment (not the first time conda env exports have screwed me). To remedy, I created a fresh environment and manually installed the deps, after which I used [conda-minify](https://github.com/jamespreed/conda-minify) to export a relaxed version.

`conda-minify --name C:\Users\Home\Anaconda3\envs\manual_env --relax -f environment.yml`

Here is that export: [environment.txt](https://github.com/Akhilesh-Gogikar/MultiStockRLTrading/files/8859174/environment.txt) (was a .yml file, but github didn't let me upload. Also I manually removed the capsule-layer line)

I needed to run this in Python 3.8, so here is my process of setting that up:
```
conda create --name multi_stock_rl_trading python=3.8
conda activate multi_stock_rl_trading
conda env update -f environment.yml
pip install git+https://github.com/leftthomas/CapsuleLayer.git@master
```
And, yay !! train.py works on GPU. Anyways, I thought I would share this to ease the pain of anyone else having the same issues.
